demo/molecule/dropdownList/demo/hoc/withStateSingle.js

```diff
@@ -23,12 +26,17 @@ const withStateSingle = BaseComponent => {
 
     render() {
       const {value} = this.state
-      const {handleSelect, props} = this
+      const {handleSelect, innerRef, props} = this
       return (
         <div>
           <pre>{JSON.stringify(this.state)}</pre>
           <div className={CLASS_DEMO_LIST}>
-            <BaseComponent {...props} value={value} onSelect={handleSelect} />
+            <BaseComponent
+              {...props}
+              innerRef={innerRef}
```

Comment:
wouldn't this work by just using regular react "ref" in order to avoid having to define innerRef in the DropdownList components?